pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Given that vagrant is so much slower than docker and cannot be run inside VMs it think that making docker scenario default would be better

Reply:
Yes, makes sense